### PR TITLE
Guardian Members Redesign

### PR DESCRIPTION
**Pull request by davidrapson**
This is now in a position to review. I've pulled out all non-visual changes I can, but the PR is still quite large. Apologies.

Some cross-browser notes: The angled-cuts across the site should be supported in Chrome, Safari and Opera (both desktop and covering iOS/Android). On Firefox/IE it is expected that these fallback to rectangular shapes.

# Screenshots

homepage
masterclasses
events
event-detail-small
brochure-page
event-details

#  QA

Just adding notes from Sean here with clarifications and/or fixes where appropriate:

- [x] ~~Are the competition pages not meant to be rebranded/redesigned?~~ Referring to pages on theguardian.com, this is OK.
- [x] ~~Should the home page have consistent treatment of the images?  Large image has angle effect but the smaller images do not~~ Intentional
- [x] Should the image on book event/masterclass page have the same treatment as the image else where?
- [x] **IE9** The 'Member' text in the logo partially falls off the edge of the banner and is not visible
- [x] ~~**IE9** The 'G Live' logo at the bottom of the page is stretched~~ This a known issue with the provider logos (also applies to third-party logos); can be fixed using the same technique that fixes the logo stretching in the header but I'll fix this in a later PR as it's an issue with the current design too.
- [x] **IE9** Optional text on Feedback page is a little close to the other elements
- [x] ~~**Safari Windows 8.1** About page White button styling inconsistent with coloured button styling~~ Safari on Windows was deprecated a number of years ago (locked to 5.1, current version 9) so a couple of visual bugs are fine.

# Release

- [ ] Awaiting final approved logo before this can go live. 

@afiore @tudorraul @rtyley

### CODE REVIEW

File: frontend/app/views/event/masterclass.scala.html [hunk 1]
```diff
@@ -78,8 +75,10 @@
             </div>
         </div>
 
-        @fragments.eventListings.listing(events,  "Sorry, no matching Masterclasses were found.", true)
-        @fragments.tier.joinListing()
+        <div class="l-constrained">
+            @fragments.eventListings.listing(events,  "Sorry, no matching Masterclasses were found.", true)
```
Comment (afiore) [inline frontend/app/views/event/masterclass.scala.html]: minor Scala readability tip: explicitly assign boolean arguments by name:

```scala
fragments.eventListings.listing(..., isFiltrable = true)
```

File: frontend/app/views/event/page.scala.html [hunk 2]
```diff
@@ -54,63 +54,57 @@ <h1 class="event-masthead__name qa-event-detail-name">@event.name.text</h1>
...
-                    <div class="event-image-credit u-event-content-width">
-                        <div class="event-image-credit__detail">
-                            @fragments.inlineIcon("caption-camera")
-                            @for(imgDescription <- imgMetadata.description) {
-                                <span class="u-align-middle">@imgDescription</span>
-                            }
-                            <span class="u-align-middle">Photograph: @imgMetadata.photographer</span>
-                        </div>
+        @for(img <- event.imgOpt if event.hasLargeImage) {
+            <div class="event-image event-image--clippath-@Random.nextInt(2).toString">
```
Comment (afiore) [inline frontend/app/views/event/page.scala.html]: You can lose the `toString` call there

File: frontend/app/views/fragments/event/item.scala.html [hunk 3]
```diff
@@ -1,8 +1,9 @@
 @(event: model.RichEvent.RichEvent, showPrice: Boolean = true)
 
-@import views.support.Dates._
+@import scala.util.Random
+@import views.support.Dates.dateRange
 
-<a href="@routes.Event.details(event.slug)" class="event-item@if(!showPrice) { event-item--shallow} @if(event.isBookable){ qa-available-event-item}">
+<a href="@routes.Event.details(event.slug)" class="event-item@if(!showPrice) { event-item--shallow} @if(event.isBookable){ qa-available-event-item} event-item--clippath-@Random.nextInt(2).toString">
```
Comment (afiore) [inline frontend/app/views/fragments/event/item.scala.html]: Same here, no need of `toString`...

**Comment by afiore** [comment]
:+1: Looking good on the Scala front.

**Comment by TesterSpike** [comment]
**IE 9+** 
* The 'Member' text in the logo partially falls off the edge of the banner and is not visible  
* The 'G Live' logo at the bottom of the page is stretched  
* 'Optional' text on Feedback page is a little close to the other elements - existing issue  

**Safari on Windows**   
* Button styling inconsistent on the patron, pricing, and about pages, some are fat and others thin  

For all other browsers the behaviour is as expected  

**General quirk**  
* The masterclass/event page small image does not have the effect applied  

**Notes**  
* Small images on home page will not have the effect to make it more noticeable else where on the site  
* Individual competition page is supplied from the main Guardian site so does not have the new styling applied

:+1:  once the quirks on IE are resolved.  Safari on windows is a small portion of our traffic.

**Comment by davidrapson** [comment]
@TesterSpike Cheers, added all these to the original description with notes.

**Comment by mattandrews** [comment]
Looks nice! :+1:

**Comment by davidrapson** [comment]
Fallback logo for launch:

screen shot 2015-08-24 at 17 14 45

**Comment by davidrapson** [comment]
screen shot 2015-08-25 at 11 58 02
screen shot 2015-08-25 at 11 58 16
screen shot 2015-08-25 at 11 58 09

**Comment by davidrapson** [comment]
Only 1 month and 2 days.

**Pull request merged**